jcl/src/java.base/share/classes/openj9/internal/foreign/abi/InternalUpcallHandler.java

```diff
@@ -22,28 +22,48 @@
  *******************************************************************************/
 package openj9.internal.foreign.abi;
 
+import java.lang.invoke.MethodHandle;
+import java.lang.invoke.MethodHandles;
+import java.lang.invoke.MethodHandles.Lookup;
+import java.lang.invoke.MethodType;
+import java.lang.invoke.WrongMethodTypeException;
 import java.util.HashMap;
 import java.util.List;
+import java.util.Set;
 
-import java.lang.invoke.MethodHandle;
-import java.lang.invoke.MethodType;
+import static java.lang.invoke.MethodHandles.empty;
+import static java.lang.invoke.MethodHandles.filterArguments;
+import static java.lang.invoke.MethodHandles.filterReturnValue;
+import static java.lang.invoke.MethodHandles.identity;
+import static java.lang.invoke.MethodHandles.insertArguments;
+import static java.lang.invoke.MethodType.methodType;
```

Comment:
IMO these static imports are not worth the extra effort required to read the code. Each of these methods is used just once or a small number of times; please just qualify those uses instead.

Reply:
Removed these static imports for **MethodHandles** (except  **MethodType.methodType** as it is used everywhere for simplicity) and directly call these method from **MethodHandles** in the related code.

Reply:
That seems reasonable.